DQMOffline/Configuration/python/autoDQM.py

```diff
@@ -1,18 +1,22 @@
-autoDQM = { 'common': ['@dcs+DQMMessageLoggerSeq+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@castor+@physics+@tau',
+autoDQM = { 'DQMMessageLogger': ['DQMMessageLoggerSeq',
+                              'PostDQMOffline',
+                              'DQMMessageLoggerClientSeq'],
+
+            'common': ['@dcs+@DQMMessageLogger+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@castor+@physics+@tau',
                         'PostDQMOffline',
-                        '@dcs+DQMMessageLoggerClientSeq+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@fed+@tau+dqmFastTimerServiceClient'],
+                        '@dcs+@DQMMessageLogger+@strip+@pixel+@tracking+@L1TMon+@hlt+@beam+@fed+@tau+dqmFastTimerServiceClient'],
 
```

Comment:
@jfernan2 this is not properly working, as far as I could see in the log file, even if it is not causing a crash

Reply:
I do not see that warning any more in your tests, so it could be ok (although not sure why)